File: platform-instances/authorization-server/src/main/java/com/sv/youapp/service/authorization/AuthApplication.java
package com.sv.youapp.service.authorization;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = {"com.sv.youapp"})
public class AuthApplication {
  public static void main(String[] args) {
    SpringApplication.run(AuthApplication.class, args);
  }
}

File: platform-instances/authorization-server/src/main/kotlin/com/sv/youapp/service/authorization/AuthApplication.kt
package com.sv.youapp.service.authorization

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.sv.youapp"])
class AuthApplication

fun main(args: Array<String>) {
    runApplication<AuthApplication>(*args)
}
